frontend/src/metabase/dashboard/components/DashCard/DashCardTabMenu/DashCardTabMenu.styled.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from "@emotion/styled";
+import type { HTMLAttributes } from "react";
+import { Menu } from "metabase/ui";
+import DashCardActionButton from "../DashCardActionButtons/DashCardActionButton";
+
+export const ItemWithMaxWidth = styled(Menu.Item)<
```

Comment:
Overriding internal component styles is error-prone and should be always avoided. For this case you could do the following:
1. Pass a styled child to unchanged `Menu.Item` and set these properties on the child.
2. Pass `component` prop on `Menu.Item`